zproject/computed_settings.py

```diff
@@ -970,6 +972,11 @@ def zulip_path(path: str) -> str:
     },
 }
 
+# Silence CryptographyDeprecationWarning spam from a dependency:
+# /srv/zulip-py3-venv/lib/python3.6/site-packages/jose/backends/cryptography_backend.py:18: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
+# TODO: Clean this up when possible after future dependency upgrades.
+warnings.filterwarnings("ignore", category=CryptographyDeprecationWarning, module="jose.*")
```

Comment:
Have you sent a PR upstream for this?

Reply:
Oh yeah, we should definitely do that.

Reply:
Yeah, will do

Reply:
Actually, fixed in https://github.com/mpdavis/python-jose/commit/b899276cc3e46f6c304f81687404294764612852 https://github.com/mpdavis/python-jose/pull/229 but not released yet